### PR TITLE
Completed prep-javascript-objects exercise

### PR DESCRIPTION
**Pull request by Edward-J-Kim**
save2

### CODE REVIEW

File: prep-javascript-objects/prep-javascript-objects-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# prep-javascript-objects-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What data types can object properties hold?
```
Comment (Zhiwutian) [inline prep-javascript-objects/prep-javascript-objects-quiz-notes.md:7]: don't forget the quiz questions

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
just need answers for the quiz questions

File: prep-javascript-objects/prep-javascript-objects-quiz-notes.md (hunk 1 above)
Comment (Edward-J-Kim) [reply prep-javascript-objects/prep-javascript-objects-quiz-notes.md:7]: answered quiz questions.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**